### PR TITLE
Specify a device support policy

### PR DESCRIPTION
**Pull request by achow101**
Specifies a policy for supporting a device as discussed in https://github.com/bitcoin-core/HWI/issues/382#issuecomment-782913737

### CODE REVIEW

File: docs/devices/index.rst [hunk 1]
```diff
@@ -56,6 +56,27 @@ The table below lists what devices and features are supported for each device.
 | Display on device screen           | ✓             | ✓             | ✓          | ✓              | ―        | ✓        | ✓       | ✓        |
 +------------------------------------+---------------+---------------+------------+----------------+----------+----------+---------+----------+
 
+Support Policy
+================
+
+For a device to be supported by HWI, it must:
+
+* Use open source firmware as much as possible
```
Comment (instagibbs) [inline docs/devices/index.rst:64]: having trouble installing sphinx on this dumb mac machine, do these nested comments look right to you?

**Review by instagibbs — APPROVED** [review]

**Comment by instagibbs** [comment]
utACK e56f8fd

File: docs/devices/index.rst (hunk 1 above)
Comment (achow101) [reply docs/devices/index.rst:64]: Apparently not. Fixed.

**Comment by achow101** [comment]
There's a preview of the docs build of this PR here: https://hwi--469.org.readthedocs.build/en/469/

**Comment by instagibbs** [comment]
utACK 6361a18

**Comment by Sjors** [comment]
utACK 6361a18

File: docs/devices/index.rst [hunk 2]
```diff
@@ -56,6 +56,34 @@ The table below lists what devices and features are supported for each device.
...
+  * A simulator/emulator is available for automated tests to be run
+  * A promise to maintain and support from the vendor:
+
+Device support may be dropped:
+
+* If promised vendor maintenance and support disappears
+
+  * If there are continuous issues with the device and the vendor has failed to provide support and updates
+
+* If the device no longer receives security devices and there are known vulnerabilities and issues
```
Comment (harding) [inline docs/devices/index.rst:85]: I think this was meant: s/security devices/security updates/

**Pull request merged**